map/benchmark_tool/benchmark_tool.pro

```diff
@@ -6,6 +6,7 @@ CONFIG -= app_bundle
 TEMPLATE = app
 
 ROOT_DIR = ../..
+# TODO(AlexZ): Remove map dependency.
```

Comment:
If benchmark tool should not depend on map probably it's worth to move it out from map directory?